Bynder/Sdk/Query/Asset/AssetUsageQuery.cs

```diff
@@ -0,0 +1,54 @@
+using Bynder.Sdk.Query.Decoder;
+using System;
+using System.Collections.Generic;
+using System.Text;
```

Comment:
I think the build should succeed if we add this:
```
using Bynder.Sdk.Api.Converters;
```

Reply:
I think so too 👍

Reply:
Just to be sure, are you planning to do this change? In that case we can merge this pull request and release a new version of the SDK.